CMakeLists.txt

```diff
@@ -99,7 +99,7 @@ elseif (EXISTS "${CMAKE_SOURCE_DIR}/.git")
         endif()
     else()
         set(PROJECT_VERSION_TWEAK 0)
-        set(VERSION_REVISION " unknown development version")
+        set(VERSION_REVISION " Culture Rework")
```

Comment:
I've just have answered some of your comments I find most important for the model itself. Other technical things could be easily resolved in case this model is in general acceptable to include into Augustus.
And thank you for taking so deep dive into this PR. We all here (players and community) really appreciate your time and efforts!